accepted/2019/support-for-memory-limits.md

```diff
@@ -29,30 +49,74 @@ The GC will more aggressive perform GCs as the GC heap grows closer to the `GCHe
 
 The GC will throw an `OutOfMemoryException` for allocations that would cause the committed heap size to exceed the `GCHeapHardLimit` memory size, even after a full compacting GC.
 
-## GC Heap Heap Minimum Size
+## GC Heap Count
 
 Using Server GC, there are multiple GC heaps created, up to one per core. This model doesn't scale well when a small memory limit is set on a machine with many cores.
 
-The minimum _reserved_ segment size per heap: 16mb
+The heap count can be set two ways:
+
+- Manually via `DOTNET_GCHeapCount`.
+- Automatically by the GC, relying on:
+  - Number of observed or configured cores.
+  - A minimum _reserved_ memory size per heap of `16 MB`.
+
+If [`DOTNET_PROCESSOR_COUNT`](https://github.com/dotnet/runtime/issues/48094) is set, including if it differs from `--cpus`, then the GC will use the ENV value for determining the maximum number of heaps to create.
 
-Example:
+Note: [.NET Framework 4.8 and 4.8.1](https://github.com/microsoft/dotnet-framework-docker/discussions/935) have the same behavior but `COMPlus_RUNNING_IN_CONTAINER` must be set. Also processor count is affected (in the same way) by `COMPlus_PROCESSOR_COUNT`.
+
+Let's look at some examples.
+
+### Memory constrained; CPU unconstrained
+
+```bash
+docker run --rm -m 256mb mcr.microsoft.com/dotnet/samples
+```
 
 * 48 core machine
-* cgroup has a 200MB memory limit
+* cgroup has a 256MB memory limit
```

Comment:
Nit: Space or no space between the number and the unit? (some places in the doc have it, some don't)

Reply:
Which do you prefer?

Reply:
Space.

Reply:
Done.